pkg/controller/resourceclaim/controller.go

```diff
@@ -384,7 +384,7 @@ func (ec *Controller) handleClaim(ctx context.Context, pod *v1.Pod, podClaim v1.
 }
 
 func (ec *Controller) syncClaim(ctx context.Context, namespace, name string) error {
-	logger := klog.LoggerWithValues(klog.FromContext(ctx), "claim", klog.KRef(namespace, name))
+	logger := klog.LoggerWithValues(klog.FromContext(ctx), "PVC", klog.KRef(namespace, name))
```

Comment:
Why this change?

Reply:
Updated based on this comment
https://github.com/kubernetes/kubernetes/pull/113584#discussion_r1107032280